File: MiniDOMWebCtrls/DMD/Utilities/ImportFromCLHandler.cs

namespace minidom.Forms
{
    public class ImportFromCLHandler : CBaseModuleHandler
    {
        public ImportFromCLHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SEdit | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SAnnotations)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Finanziaria.CCQSPDOfferteCursor();
        }
    }
}

File: MiniDOMWebCtrls/DMD/Utilities/ImportFromCLHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Finanziaria
Imports minidom.Sistema
Imports minidom.Anagrafica

Namespace Forms

    Public Class ImportFromCLHandler
        Inherits CBaseModuleHandler


        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SAnnotations)
            
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New CCQSPDOfferteCursor
        End Function


    End Class


End Namespace